dateparser/utils/__init__.py

```diff
@@ -139,6 +139,26 @@ def get_last_day_of_month(year, month):
     return calendar.monthrange(year, month)[1]
 
 
+def get_previous_leap_year(year):
+    return _get_leap_year(year, future=False)
+
+
+def get_next_leap_year(year):
+    return _get_leap_year(year, future=True)
+
+
+def _get_leap_year(year, future):
+    """
+    Iterate through previous or next years until it gets a valid leap year
+    This is performed to avoid missing or including centurial leap years
+    """
+    difference = 1 if future else -1
```

Comment:
:lipstick: I’m unsure about this one, but since I’m going full style suggestions, 1 more should not hurt :slightly_smiling_face: 

```suggestion
    step = 1 if future else -1
```

Reply:
hahaha it's ok